aws-event-sources-operator/bundle/0.0.1/aws-event-sources-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,277 @@
+          resources:
+          - servicemonitors
+          verbs:
+          - get
+          - create
+        - apiGroups:
+          - apps
+          resourceNames:
+          - aws-event-sources-operator
+          resources:
+          - deployments/finalizers
+          verbs:
+          - update
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          verbs:
+          - get
+        - apiGroups:
+          - ""
+          resources:
+          - services
+          - services/finalizers
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - apps
+          resources:
+          - replicasets
+          - deployments
+          verbs:
+          - get
+        - apiGroups:
+          - sources.triggermesh.com
+          resources:
+          - '*'
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - apiextensions.k8s.io
+          resources:
+          - customresourcedefinitions
+          verbs:
+          - '*'
+        serviceAccountName: aws-event-sources-operator
+      deployments:
+      - name: aws-event-sources-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: aws-event-sources-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: aws-event-sources-operator
+            spec:
+              containers:
+              - env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: aws-event-sources-operator
+                - name: AWS_EVENT_SOURCES_CONTROLLER_IMAGE
+                  value: gcr.io/triggermesh/aws-event-sources-controller:v0.3.0
+                - name: AWSCODECOMMIT_ADAPTER_IMAGE
+                  value: gcr.io/triggermesh/awscodecommitsource:v0.3.0
+                - name: AWSCOGNITOIDENTITY_ADAPTER_IMAGE
+                  value: gcr.io/triggermesh/awscognitoidentitysource:v0.3.0
+                - name: AWSCOGNITOUSERPOOL_ADAPTER_IMAGE
+                  value: gcr.io/triggermesh/awscognitouserpoolsource:v0.3.0
+                - name: AWSDYNAMODB_ADAPTER_IMAGE
+                  value: gcr.io/triggermesh/awsdynamodbsource:v0.3.0
+                - name: AWSKINESIS_ADAPTER_IMAGE
+                  value: gcr.io/triggermesh/awskinesissource:v0.3.0
+                - name: AWSSNS_ADAPTER_IMAGE
+                  value: gcr.io/triggermesh/awssnssource:latest
```

Comment:
Same comment about SNS being part of the releases.

Reply:
fixed